portfolio/src/main/webapp/script.js

```diff
@@ -37,3 +37,10 @@ function getHellofromFetchedPractice() {
     document.getElementById('fetch-container').innerText = quote;
   });
 }
+
+
+function getCommentInForm() {
+  fetch('/data').then(response => response.text()).then((quote) => {
```

Comment:
nit: it is not returning a quote but instead a comment now, could update the variable name to make it clearer :)